sample-apps/SampleAppSpec.md

```diff
@@ -0,0 +1,233 @@
+## Standardized Sample App Spec - README
+
+### Top level Requirements
+
+#### Where the Sample Apps should live
+
+Each sample app that follows the requirements below should be stored in the `sample-apps` folder in https://github.com/aws-observability/aws-otel-community.
+The matrix that exists in the README file in that directory should be updated with the sample apps in conformance to this document. The matrix must additionally display the status of Metrics Logs and Traces. (Stable or Unstable)
+
+#### How the Sample Apps should work
+
+Each sample app should emit the same Metrics, Logs and Traces. If a sample app cannot emit the mentioned, then that will noted in that sample app’s documentation.
```

Comment:
```suggestion
Each sample app must emit the same Metrics, Logs and Traces. If a sample app cannot emit the mentioned, then that will noted in that sample app’s documentation.
```
Should or must?

Reply:
I think must would better enforce the standard